app/scripts/controllers/overview.js

```diff
@@ -1349,7 +1349,9 @@ function OverviewController($scope,
       updateQuotaWarnings();
     }, {poll: true, pollInterval: DEFAULT_POLL_INTERVAL}));
 
-    if (SERVICE_CATALOG_ENABLED) {
+    var canI = $filter('canI');
+    // The canI check on watch should be temporary until we have a different solution for handling secret parameters
```

Comment:
Curious (my own ignorance) how changing what we do for handling secret parameters will help with this watch on `instances`?  Not saying the comment needs more necessarily, I just don't know that I follow.